### PR TITLE
improvement(schema changes): Add LWT load

### PR DESCRIPTION
**Pull request by yarongilor**
In order to cover LWT handling parallel schema changes
	and topology changes nemesis.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
Tested in: https://jenkins.scylladb.com/job/scylla-staging/job/yarongilor/job/longevity-alternator-dbg/11/

File: test-cases/longevity/longevity-parallel-schema-changes-12h.yaml [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-             "scylla-bench -workload=sequential -mode=read  -replication-factor=3 -partition-count=50 -clustering-row-count=5000 -partition-offset=121 -clustering-row-size=uniform:4096..8192   -concurrency=100 -connection-count=100 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=700m -validate-data"
+             "scylla-bench -workload=sequential -mode=read  -replication-factor=3 -partition-count=50 -clustering-row-count=5000 -partition-offset=121 -clustering-row-size=uniform:4096..8192   -concurrency=100 -connection-count=100 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=700m -validate-data",
+             "cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(select=1)' cl=SERIAL duration=700m -mode native cql3 -rate threads=10"
```
Comment (roydahan) [inline test-cases/longevity/longevity-parallel-schema-changes-12h.yaml:15]: If you don't have nemesis_during_prepare, the LWT here is "select" only, it doesn't really exercise it.

Comment (yarongilor) [reply test-cases/longevity/longevity-parallel-schema-changes-12h.yaml:15]: ok, @roydahan , added an `update` load as well. (BTW i'm not sure whether the original issue is related to read/write/none (schema-only)

**Comment by fgelcer** [comment]
@yarongilor , what is the status of this PR? if we don't need it, please close it, and if we do want to move it forward, change it to ready for review.

**Comment by yarongilor** [comment]
Closing for now since current tests coverage (LWT schema changes and parallel alter-table nemesis test) is sufficient.

**Pull request closed without merging**